### PR TITLE
[Messenger] Document the debug option of Amazon SQS

### LINKED ISSUE #14533: [Messenger][SQS] Make sure one can enable debug logs
| Q            | A
| ------------ | ---
| Feature PR   | symfony/symfony#38998
| PR author(s) | @Nyholm
| Merged in    | 5.3-dev

### PR DESCRIPTION
**Pull request by javiereguiluz**
Fixes #14533.

### CODE REVIEW

File: messenger.rst [hunk 1]
```diff
@@ -1342,6 +1342,8 @@ The transport has a number of options:
 ``auto_setup``          Whether the queue should be created     ``true``
                         automatically during send / get.
 ``buffer_size``         Number of messages to prefetch          9
+``debug``               If ``true`` it logs all HTTP requests   false
```
Comment (OskarStark) [inline messenger.rst]:
```suggestion
``debug``               If ``true`` it logs all HTTP requests   ``false``
```

like above

**Review by OskarStark — CHANGES_REQUESTED** [review]
just a small one

File: messenger.rst (hunk 1 above)
Comment (javiereguiluz) [reply messenger.rst]: Done!

**Review by OskarStark — APPROVED** [review]

**Comment by OskarStark** [comment]
Thank you Javier.

**Comment by Nyholm** [comment]
Oh, I forgot about this. 

Thank you @javiereguiluz

**Pull request merged**